tests/test_flake8_spellcheck.py

```diff
@@ -104,6 +104,12 @@ def test_flake8_pragma_spaces(self, flake8dir):
             "./example.py:1:14: E262 inline comment should start with '# '"
         ]
 
+    # Regression test for github.com/MichaelAquilina/flake8-spellcheck/issues/34
+    def test_empty_comment(self, flake8dir):
+        flake8dir.make_example_py("#")
```

Comment:
could you expand this a bit more? Maybe something like

```python
flake8dir.make_example_py("""
    # the line below should not fail
    #
    # hello world
""")
```

Reply:
sure!